File: Facil.py
# FÁCIL
# Crie uma lista de cores, e percorra cada elemento de trás pra frente da lista exibindo o índice e o valor


cores = ['vermelho', 'azul', 'laranja', 'preto', 'branco', 'cinza', 'verde', 'amarelo']

for cor in cores[7::-1]: #inicia no fim da lista e percorre de trás pra frente
    print(f'A cor é {cor} e o índice é {cores.index(cor)}') # cores.index(cor) - pega o indice da cor atual

File: facil.js
// ler um array de trás pra frente e exibir o indice e seu valor

const cores = ['vermelho', 'azul', 'laranja', 'preto', 'branco', 'cinza', 'verde', 'amarelo']

for (let i = cores.length - 1;  i >= 0 ; i --) {
    console.log(`Indice: ${i} | Cor: ${cores[i]}`);
}
